src/main/java/com/github/hcsp/collection/Main.java

```diff
@@ -5,15 +5,17 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Arrays;
+import java.util.ArrayList;
 
 
 
 public class Main {
     // 请编写一个方法，获得a和b集合中的公共元素。
     public static Set<Person> commonElementsIn(List<Person> a, List<Person> b) {
         Set<Person> commonSet = new HashSet<>();
-        a.retainAll(b);
-        commonSet.addAll(a);
+        ArrayList commonList = new ArrayList(a);
```

Comment:
@blindpirate i got an `java.lang.ClassCastException` when I  cast the `List` to `ArrayList` so I use  `new ArrayList()`
 instead , any advice?